--- /etc/X11/imwheel/startup.conf.orig	2014-05-13 12:38:39.696013946 +0200
+++ /etc/X11/imwheel/startup.conf	2014-05-13 12:39:19.619647494 +0200
@@ -1,7 +1,7 @@
 # Configuration file for setting imwheel startup parameters.
 
 # Set this to "1" to have imwheel start along with your X session.
-IMWHEEL_START=0
+IMWHEEL_START=1
 
 # Specify the command line parameters to pass to imwheel when it is started
 # along with your X session (above). Simply uncomment the bottom line, and
